### PR TITLE
sweep of last bits of internationalization for displayed text

### PR DESCRIPTION
**Pull request by EugeneJoe**

### CODE REVIEW

**Review by utunga — APPROVED** [review]
👍  LGTM - love that you've preserved the Swedish where possible

File: sb.web.app/src/components/AdminSection/Members/member.vue [hunk 1]
```diff
@@ -33,14 +33,20 @@ do not match the equivalent of the database.
...
+import { EXPRESS_URL, getUserSaldo } from '@/serverFetch'
 
 export default {
 
   props: {
     listingObj: Object
   },
   methods: {
+    async getBalance () {
+      const saldo = await getUserSaldo(this.listingObj)
```
Comment (utunga) [inline sb.web.app/src/components/AdminSection/Members/member.vue:45]: 👍

**Comment by utunga** [comment]
i tried to flag @gitguardian's thing as a false positive but it's still moaning about it. GitGuardian can get in the sea.

**Comment by utunga** [comment]
this is all goods @EugeneJoe  we should merge eh?

**Comment by EugeneJoe** [comment]
yup.
Think it looks good.
Working on another.

**Pull request merged**